src/get-variables.js

```diff
@@ -1,7 +1,7 @@
 import stripComments from 'strip-json-comments';
 
 export default function getVariables(content) {
-  const variableRegex = /\$(.+):\s+(.+);/;
+  const variableRegex = /\$(.+):\s+(.+);?/;
```

Comment:
Oh yeah! It's been so long since I wrote this code. My bad 😅
In that case this looks great!